idd/Energy+.idd.in

```diff
@@ -45619,7 +45615,131 @@ Coil:Heating:DX:VariableSpeed,
         \note quadratic curve = a + b*ffa + c*ffa**2
         \note cubic curve = a + b*ffa + c*ffa**2 + d*ffa**3
         \note ffa = Fraction of the full load Air Flow
+						
+COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE,
+	   \memo This object is used for air-source integrated heat pump, a collection of its working modes.
+  A1, \field Name
+       \required-field
+       \type alpha
+       \reference IntegratedHeatPumps
+       \note Unique name for this instance of an air-source integrated heat pump.
+  A2, \field Indoor Air Cooling Coil Inlet Node Name
+       \required-field
+       \type alpha
+  A3, \field Indoor Air Heating Coil Inlet Node Name
+       \required-field
+       \type alpha
+  A4, \field Indoor Air Outlet Node Name
+       \required-field
+       \type alpha
+  A5, \field Condenser Water Inlet Node
+       \required-field
+       \type alpha
+  A6, \field Condenser Water Outlet Node
+       \required-field
+       \type alpha
+  A7, \field Water Node to Monitor the Supply Hot Water Flow
```

Comment:
That feels like an odd input field name.  Maybe `Supply Hot Water Flow Sensing Node Name` or something of that nature?